packages/reporter/src/test/test.tsx

```diff
@@ -193,7 +193,7 @@ class Test extends Component<TestProps> {
       )
     }
 
-    if (this.props.studioEnabled) {
+    if (this.props.studioEnabled && !appState.studioActive) {
```

Comment:
Seems this is actually breaking some tests in reporter - do we need this check at all?